### ISSUE TITLE
Difficulty backwards for Mile2 certs

### ISSUE THREAD

**Issue by sawft99** [PaulJerimy/SecCertRoadmapHTML#57]
I noticed for Mile2 certs you have C)ISSM lower than C)ISSO which gives the impression that it is easier and I know that the difficulty is semi vague. However, I notice that Mile2 lists C)ISSM after C)ISSO making it seem like that is the progression they recommend.

https://www.mile2.com/information_systems_security_auditor_outline/

Same for C)ISSM and C)ISMS-LA/LI

https://www.mile2.com/cisms-la-li-outline/

C)CSA and C)TIA near the bottom as well when they are listed as more difficult certs

**Comment by sinecurelife** [comment]
Hey Sawft99, thanks for the insight. When I initially reviewed Mile2 certs their roadmap was different. I believe I didn't use their roadmap and instead opened all the certs and compared their syllabi.

I'm giving those four certs another look today, especially since C)ISSO got an update last year.

**Comment by sinecurelife** [comment]
I have moved C)ISSM up two slots to be just under C)ISSO. While C)ISSM is after C)ISSO on the Mile2 roadmap, it covers a smaller more managerial body of knowledge. If C)ISSM is taken by itself it appears to offer less KSAs than C)ISSO would alone. C)ISSO is not required to take C)ISSM.

I have also added the CM) ISSO certification along with the other two Mile2 Master certifications. These certifications require you take the whole series of exams to earn and in the case of CM) ISSO, better encapsulates the value of doing both C)ISSO and C)ISSM.

This change will be reflected in tomorrow's push
